test/api/v4/faq/GET-faq.test.js

```diff
@@ -52,18 +38,5 @@ describe('GET /faq', () => {
           message: i18n.t('invalidReqParams'),
         });
     });
-    it('falls back to "web" description if there is no description for specified platform', async () => {
```

Comment:
Shouldn't we retain this behavior? So if we get a request for an "ios" question, we just supply the web one?

Reply:
I talked to @saraolson about this and they're pulling the web questions into mobile, so there may not be a need for this. That was my thinking when I removed this part of the test code.